### PR TITLE
[fix] #3360: Fix tps benchmark

### LINKED ISSUE #3360: [BUG]: Fix tps benchmark

### PR DESCRIPTION
**Pull request by Erigara**
## Description

- fix benchmark
- add missing ports

### Linked issue

Closes #3360 

### Benefits

- benchmark works
- tests work more stable

### Checklist

- [x] I've read `CONTRIBUTING.md`
- [x] I've used the standard signed-off commit format (or will squash just before merging)
- [x] All applicable CI checks pass (or I promised to make them pass later)
- [x] (optional) I've written unit tests for the code changes
- [x] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

File: client/benches/tps/utils.rs [hunk 1]
```diff
@@ -222,15 +222,16 @@ impl MeasurerUnit {
         let submitter = self.client.clone();
         let interval_us_per_tx = self.config.interval_us_per_tx;
         let instructions = self.instructions();
-        let account_id = account_id(self.name);
+        let alice_id = <Account as Identifiable>::Id::from_str("alice@wonderland")
+            .expect("Failed to parse account id");
```
Comment (mversic) [inline client/benches/tps/utils.rs:226]: Is this reverted on purpose?

Comment (Erigara) [reply client/benches/tps/utils.rs:226]: Yes, it is source of the problem. Because we want to submit transactions by alice.

**Review by appetrosyan — APPROVED** [review]

**Comment by appetrosyan** [comment]
@BAStos525 We should set up a CI check for benchmarks.

File: client/benches/tps/utils.rs (hunk 1 above)
Comment (mversic) [reply client/benches/tps/utils.rs:226]: I see now, my mistake. I thought it resolves to `alice@wonderland`

**Review by mversic — APPROVED** [review]

**Pull request merged**